source/renderer/app/api/api.js

```diff
@@ -722,6 +728,40 @@ export default class AdaApi {
     }
   };
 
+  createAddress = async (
+    request: CreateByronWalletAddressRequest
+  ): Promise<Address> => {
+    logger.debug('AdaApi::createAddress called', {
+      parameters: filterLogData(request),
+    });
+    const {
+      accountIndex,
+      walletId,
+      spendingPassword: passwordString,
+    } = request;
+    const spendingPassword = passwordString
+      ? encryptPassphrase(passwordString)
```

Comment:
@tomothespian there is no encryption required here - we will be sending spending password in plain text format (same as for other endpoints).

Reply:
@nikolaglumac thanks, I will remove that